### PR TITLE
Rename `AppendEncoded()` to `AppendHtml()`

### PR DESCRIPTION
**Pull request by dougbu**
- aspnet/Mvc#3225, 1 of 3

### CODE REVIEW

**Comment by dougbu** [comment]
@kichalla

File: src/Microsoft.AspNet.Html.Abstractions/IHtmlContentBuilder.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public interface IHtmlContentBuilder : IHtmlContent
         /// </summary>
         /// <param name="encoded">The HTML encoded <see cref="string"/> to append.</param>
         /// <returns>The <see cref="IHtmlContentBuilder"/>.</returns>
-        IHtmlContentBuilder AppendEncoded(string encoded);
+        IHtmlContentBuilder AppendHtml(string encoded);
```
Comment (kichalla) [inline src/Microsoft.AspNet.Html.Abstractions/IHtmlContentBuilder.cs]: Should the documenation and parameter name change? example: `AppendHtml("<b>")` ...the string is not encoded but raw html...
Same comment in all other relevant places.

Comment (dougbu) [inline src/Microsoft.AspNet.Html.Abstractions/IHtmlContentBuilder.cs]: The parameter name is entirely correct and a good counterpoint to the `unencoded` parameter in `Append()`. In your example `"<b>"` is as encoded as it's going to get.

File: src/Microsoft.Extensions.BufferedHtmlContent.Sources/BufferedHtmlContent.cs [hunk 2]
```diff
@@ -48,7 +48,7 @@ public IHtmlContentBuilder Append(IHtmlContent htmlContent)
         /// </summary>
         /// <param name="value">The HTML encoded <c>string</c> to be appended.</param>
         /// <returns>A reference to this instance after the Append operation has completed.</returns>
-        public IHtmlContentBuilder AppendEncoded(string value)
+        public IHtmlContentBuilder AppendHtml(string value)
```
Comment (dougbu) [inline src/Microsoft.Extensions.BufferedHtmlContent.Sources/BufferedHtmlContent.cs]: ... in fact I'm thinking of renaming this parameter.

**Comment by dougbu** [comment]
@kichalla updated to fix parameter names, `SetContentEncoded()` name and doc comments.

File: src/Microsoft.Extensions.BufferedHtmlContent.Sources/BufferedHtmlContent.cs [hunk 3]
```diff
@@ -26,9 +26,9 @@ internal class BufferedHtmlContent : IHtmlContentBuilder
         /// </summary>
         /// <param name="value">The <c>string</c> to be appended.</param>
```
Comment (dougbu) [inline src/Microsoft.Extensions.BufferedHtmlContent.Sources/BufferedHtmlContent.cs]: `xml-docs-test` missed this. will fix.

**Comment by dougbu** [comment]
@kichalla CI builds are failing because Coherence hasn't made recent Common changes available. This is ready to go.

**Comment by kichalla** [comment]
:shipit:

**Comment by dougbu** [comment]
bcb56bd

**Pull request closed without merging**